### PR TITLE
feature(ui-ux): added create proposal page

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Sample Links & Screenshots:

Link:

<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on multiple web browsers
- [ ] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [ ] No console errors
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: src/pages/on-chain-governance/_components/GettingStartedInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { MdInfo } from "react-icons/md";
+
+export function GettingStartedInfo(): JSX.Element {
+  const link = "https://github.com/DeFiCh/dfips";
```
Comment (kyleleow) [inline src/pages/on-chain-governance/_components/GettingStartedInfo.tsx:5]:
```suggestion
  const link = "https://github.com/DeFiCh/dfips/issues";
```

File: src/pages/on-chain-governance/_components/SubmitProposal.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+interface SubmitProposalProps {
+  command: string;
+}
+export function SubmitProposal({ command }: SubmitProposalProps): JSX.Element {
+  return (
+    <>
+      <span className="text-gray-600 dark:text-gray-100 text-sm md:text-base">
+        To finalize your submission on-chain, copy the generated command line,
+        and paste it in the CLI located in your full node wallet. If you have
+        not downloaded a full node wallet,
```
Comment (veralygit) [inline src/pages/on-chain-governance/_components/SubmitProposal.tsx:13]:
```suggestion
        not downloaded a full node wallet,&nbsp;
```

File: src/pages/on-chain-governance/_components/SubmitProposal.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <a
+          className="text-[#4A72DA]"
```
Comment (veralygit) [inline src/pages/on-chain-governance/_components/SubmitProposal.tsx:15]:
```suggestion
          className="text-[#4A72DA] underline"
```

File: src/pages/on-chain-governance/_components/SubmitProposal.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          href="https://defichain.com/downloads"
+          target="_blank"
+          rel="noreferrer"
+        >
+          &nbsp;download here
+        </a>
+      </span>
+      <div className="flex flex-row mt-6 mb-2 md:mb-4 bg-blue-100 rounded py-3 px-4 items-center">
+        <span className="text-[#4A72DA] break-all">{command}</span>
```
Comment (veralygit) [inline src/pages/on-chain-governance/_components/SubmitProposal.tsx:24]:
```suggestion
                <TextTruncate
          className="text-[#4A72DA] whitespace-nowrap"
          text={command}
          width="w-[542px]"
        />
```

Comment (veralygit) [reply src/pages/on-chain-governance/_components/SubmitProposal.tsx:24]: As discuss showing the output as full command might be better

File: src/pages/on-chain-governance/_components/SubmitProposal.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        To finalize your submission on-chain, copy the generated command line,
+        and paste it in the CLI located in your full node wallet. If you have
+        not downloaded a full node wallet,&nbsp;
+        <a
+          className="text-[#4A72DA] underline"
+          href="https://defichain.com/downloads"
+          target="_blank"
+          rel="noreferrer"
+        >
+          &nbsp;download here
```
Comment (veralygit) [inline src/pages/on-chain-governance/_components/SubmitProposal.tsx:20]:
```suggestion
          download here
```

File: src/pages/on-chain-governance/_components/SubmitProposal.tsx (hunk 4 above)
Comment (veralygit) [inline src/pages/on-chain-governance/_components/SubmitProposal.tsx:24]:
```suggestion
        <span className="text-[#4A72DA] break-all flex grow">{command}</span>
```

screen width for 1550px copy icon wouldnt be at the end 
image

File: src/pages/on-chain-governance/_components/ReviewProposal.tsx [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+              </div>
+            </div>
+          </>
+        )}
+      </div>
+      <div className="mt-10 md:mt-12 text-center">
+        <button
+          type="button"
+          onClick={onClick}
+          className="w-full py-3 border border-primary-50 rounded-sm font-medium text-primary-500 bg-primary-50 mb-3"
```
Comment (veralygit) [inline src/pages/on-chain-governance/_components/ReviewProposal.tsx:127]:
```suggestion
          className="w-full py-3 border border-primary-50 rounded-sm font-medium text-primary-500 bg-primary-50 mb-3 hover:bg-primary-100 hover:border-primary-100"
```

**Review by veralygit — APPROVED** [review]

File: src/pages/on-chain-governance/index.page.tsx [hunk 7]
```diff
@@ -126,7 +126,7 @@ export default function OnChainGovernancePage(props) {
               <Button
                 label={`Submit a proposal`.toUpperCase()}
                 testId="OnChainGovernance.SubmitProposalButton"
-                onClick={() => {}}
+                onClick={() => router.push("on-chain-governance/create")}
```
Comment (kyleleow) [inline src/pages/on-chain-governance/index.page.tsx:129]: need to pass the network query too if exist on current url

File: src/pages/on-chain-governance/index.page.tsx [hunk 8]
```diff
@@ -270,7 +270,7 @@ export default function OnChainGovernancePage(props) {
             <Button
               label={`Submit a proposal`.toUpperCase()}
               testId="OnChainGovernance.SubmitProposalButton"
-              onClick={() => {}}
+              onClick={() => router.push("on-chain-governance/create")}
```
Comment (kyleleow) [inline src/pages/on-chain-governance/index.page.tsx:273]: need to pass the network query too if exist on current url

File: src/pages/on-chain-governance/_components/SubmitProposal.tsx [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+        <span className="text-[#4A72DA] break-all flex grow">{command}</span>
+        <CopyButton
+          className="ml-2"
+          content={command}
+          iconsClass="text-[#4A72DA] h-6 w-6"
+          buttonClass="cursor-pointer outline-none bg-transparent dark:bg-transparent border-none"
+        />
+      </div>
+      <span className="text-orange-600 text-xs md:text-sm">
+        Command line submitted into CLI will not be editable. Please check your
```
Comment (kyleleow) [inline src/pages/on-chain-governance/_components/SubmitProposal.tsx:33]:
```suggestion
        Command submitted into defi-cli will not be editable. Please check your
```

File: src/pages/on-chain-governance/create/index.page.tsx [hunk 10]
```diff
@@ -0,0 +1,426 @@
...
+        <Breadcrumb
+          items={[
+            {
+              path: "/on-chain-governance",
+              name: "Proposal",
+            },
+            {
+              path: `/on-chain-governance/create`,
+              name: "Create Proposal",
+              canonical: true,
```
Comment (kyleleow) [inline src/pages/on-chain-governance/create/index.page.tsx:160]: to add `isCurrentPath: true,`, current path has a different text color

File: src/pages/on-chain-governance/_components/DisclosureComponent.tsx [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+}: PropsWithChildren<{
+  title: string;
+  isOpen: boolean;
+  isCompleted?: boolean;
+  onEdit?: () => void;
+}>) {
+  return (
+    <div className="py-5 px-6 md:px-10 rounded-lg border border-gray-200 dark:border-gray-700">
+      <Disclosure defaultOpen>
+        <div className={classNames({ "py-1 md:py-5": isOpen })}>
```
Comment (kyleleow) [inline src/pages/on-chain-governance/_components/DisclosureComponent.tsx:21]:
```suggestion
        <div className={classNames({ "py-5": isOpen })}>
```

File: src/pages/on-chain-governance/create/index.page.tsx [hunk 12]
```diff
@@ -0,0 +1,426 @@
...
+          ]}
+        />
+        <h1 className="text-2xl	md:text-4xl font-semibold mt-2 mb-6 text-gray-900 dark:text-gray-100">
+          Create Proposal
+        </h1>
+        <div className="flex flex-col space-y-2">
+          <GettingStartedInfo />
+          {/* Step 1 */}
+          <DisclosureComponent
+            title="Step 1: Proposal Details"
```
Comment (kyleleow) [inline src/pages/on-chain-governance/create/index.page.tsx:171]:
```suggestion
            title="Step 1: Proposal details"
```

File: src/pages/on-chain-governance/_components/DisclosureComponent.tsx (hunk 11 above)
Comment (fullstackninja864) [reply src/pages/on-chain-governance/_components/DisclosureComponent.tsx:21]: Thanks for pointing out, Design is inconsistent for this

**Pull request merged**